bundles/org.pitest.pitclipse.launch/src/org/pitest/pitclipse/launch/config/LaunchConfigurationWrapper.java

```diff
@@ -260,7 +261,13 @@ private int getThreadCount() throws CoreException {
         return projectFinder.getProjects(this);
     }
 
-    private List<String> getMutators() {
-        return ImmutableList.of(pitConfiguration.getMutators());
+    private List<String> getMutators() throws CoreException {
+        final String mutators;
+        if (launchConfig.hasAttribute(MUTATORS)) {
+            mutators = launchConfig.getAttribute(MUTATORS, "");
+        } else {
+            mutators = pitConfiguration.getMutators();
+        }
+        return ImmutableList.of(mutators);
```

Comment:
Since we're there and since I'm already removing Guava from the codebase, please use standard List. In any case, this will always be a singleton List, right? Do we still need a List?